metadata-io/src/main/java/com/linkedin/metadata/search/cache/CachedEntityLineageResult.java

```diff
@@ -1,11 +1,10 @@
 package com.linkedin.metadata.search.cache;
 
-import com.linkedin.metadata.graph.EntityLineageResult;
 import lombok.Data;
 
 
 @Data
 public class CachedEntityLineageResult {
-  private final EntityLineageResult entityLineageResult;
+  private final String entityLineageResult;
```

Comment:
we are serializing this result? interesting

Reply:
Yeah, this is essentially a DTO wrapper around the result to be able to include the timestamp (and anything else we want to add in later) onto the cached value so any RecordTemplate used within it has to be serialized